### PR TITLE
docs: added pub.green badges to Readme

### PR DESCRIPTION
**Pull request by aagarwal1012**
Reference: https://pub.green/packages/animated_text_kit

### CODE REVIEW

**Review by SirusCodes — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -17,6 +17,19 @@
     <img src="https://img.shields.io/github/workflow/status/aagarwal1012/Animated-Text-Kit/CI?logo=github"
       alt="Build Status" />
   </a>
+  <a href="https://pub.green/packages/animated_text_kit">
```
Comment (awhitford) [inline README.md:20]: `href` should be `https://pub.green/packages/animated_text_kit#channel-stable`

File: README.md [hunk 2]
```diff
@@ -17,6 +17,19 @@
...
+    <img src="https://img.shields.io/endpoint?url=https://pub.green/packages/animated_text_kit/badge?channel=stable&style=flat"
+      alt="Latest compatibility result for Stable channel" />
+  </a>
+  <a href="https://pub.green/packages/animated_text_kit">
```
Comment (awhitford) [inline README.md:24]: `href` should be `https://pub.green/packages/animated_text_kit#channel-beta`

File: README.md [hunk 3]
```diff
@@ -17,6 +17,19 @@
...
+    <img src="https://img.shields.io/endpoint?url=https://pub.green/packages/animated_text_kit/badge?channel=beta&style=flat"
+      alt="Latest compatibility result for Beta channel" />
+  </a>
+  <a href="https://pub.green/packages/animated_text_kit">
```
Comment (awhitford) [inline README.md:28]: `href` should be `https://pub.green/packages/animated_text_kit#channel-dev`

**Comment by aagarwal1012** [comment]
@awhitford, #205 should address your comments.

**Pull request merged**